### PR TITLE
Create 2024-09-17-Reflections on GCC2024

### PR DESCRIPTION
**Pull request by GarethPrice-Aus**
To add link to external blog post at:
https://www.biocommons.org.au/blogs/gcc2024

### CODE REVIEW

**Comment by GarethPrice-Aus** [comment]
Starting new blog post to promote: https://www.biocommons.org.au/blogs/gcc2024

**Review by GarethPrice-Aus — COMMENTED** [review]
@PatCapon39 Can you approve this?

**Comment by PatCapon39** [comment]
Nope I don't have approval permissions

File: content/blog/2024-09-17-Reflections on GCC2024 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (bgruening) [inline content/blog/2024-09-17-Reflections on GCC2024:1]: @GarethPrice-Aus this file is empty?

**Comment by GarethPrice-Aus** [comment]
yeah, I stuffed up on making a folder and then GithubDesktop isn't helping me fix. 
Please reject so I can start again

**Comment by bgruening** [comment]
Just change this file in an additional PR.

See here for external URLs: https://github.com/galaxyproject/galaxy-hub/blob/bbc9bede5b3f5a7b198b5d595252df0fc51c93b2/content/news/2021-11-gtn-api/index.md?plain=1#L5

I invented @PatCapon39 to this repo here.

**Comment by PatCapon39** [comment]
Thanks @bgruening!

**Pull request merged**